packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -233,6 +249,27 @@ export async function activate(context: vscode.ExtensionContext) {
     sfdxProjectOpened = files && files.length > 0;
   }
 
+  // remove before merging
```

Comment:
I'm commenting for this as a reminder so my comment gets hidden when this is removed before merging.